land_claims.civmap.json

```diff
@@ -79,7 +83,11 @@
+{"name":"Bloom\n(Ashelor)","shortname":"Bloom","color":"#6c1282","declutter":2,"x":-1945,"z":-2750,"id":"a84c25e3-76eb-4b87-b1ca-f9dbdd59b892","polygon":[[[-1675,-2302],[-1647,-2327],[-1652,-2358],[-1628,-2379],[-1626,-2398],[-1601,-2402],[-1568,-2383],[-1543,-2361],[-1429,-2406],[-1380,-2371],[-1289,-2372],[-1220,-2472],[-1170,-2472],[-1077,-2514],[-1047,-2544],[-1045,-2578],[-1094,-2606],[-1116,-2609],[-1148,-2587],[-1204,-2710],[-1208,-2884],[-1382,-2881],[-1420,-2937],[-1458,-2953],[-1528,-3045],[-1428,-3145],[-1478,-3447],[-1548,-3563],[-1498,-3637],[-1454,-3671],[-1433,-3706],[-1472,-3751],[-1504,-3752],[-1544,-3729],[-1609,-3738],[-1654,-3772],[-1774,-3756],[-1828,-3729],[-1907,-3635],[-1940,-3523],[-1914,-3423],[-2060,-3249],[-2096,-3242],[-2144,-3189],[-2184,-3189],[-2224,-3224],[-2384,-3232],[-2386,-3173],[-2338,-3090],[-2231,-2980],[-2300,-2905],[-2373,-2933],[-2434,-2923],[-2495,-2858],[-2603,-2852],[-2638,-2830],[-2764,-2832],[-2800,-2726],[-2802,-2632],[-2764,-2579],[-2766,-2538],[-2704,-2476],[-2628,-2418],[-2625,-2160],[-1721,-2161],[-1703,-2277],[-1691,-2295]]]},
+{"name":"Caledonia","shortname":"Caledonia","color":"#15316a","declutter":1,"x":8800,"z":850,"id":"5d940a0b-1bf4-4862-8397-ec565c38ef30","polygon":[[[9515,-266],[9541,-284],[9572,-333],[9598,-362],[9624,-365],[9668,-330],[9667,-299],[9655,-288],[9620,-253],[9617,-249],[9615,-230],[9638,-192],[9582,-159],[9581,-74],[9688,44],[9620,676],[9750,835],[9831,787],[9900,871],[9893,904],[9913,941],[9901,987],[9930,1028],[9850,1116],[9825,1173],[9829,1225],[9864,1260],[9927,1249],[9969,1273],[9913,1315],[9845,1313],[9823,1348],[9839,1398],[9900,1502],[9739,1594],[9631,1580],[9390,1494],[8878,1680],[8764,1616],[8750,1496],[8708,1482],[8634,1498],[8614,1504],[8530,1418],[8532,1378],[8458,1340],[8420,1336],[8326,1394],[8220,1448],[8092,1384],[8124,1296],[8156,1282],[8152,1250],[8124,1228],[8048,1232],[8032,1236],[7974,1178],[7974,1136],[7948,1124],[7880,1124],[7828,1180],[7752,1182],[7736,1182],[7716,1152],[7664,1152],[7608,1198],[7564,1260],[7480,1244],[7436,1160],[7460,1128],[7436,1052],[7437,996],[7497,995],[7544,1019],[7585,992],[7569,923],[7647,813],[7703,833],[7755,795],[7789,826],[7810,872],[7865,827],[7879,823],[7927,850],[7983,812],[7984,774],[8018,730],[8016,678],[7970,626],[7928,610],[7886,554],[7840,526],[7800,474],[7718,452],[7736,372],[7778,312],[7832,294],[7870,250],[7908,254],[7920,294],[7984,314],[8030,346],[8232,-384],[8472,-548],[8768,-364],[8984,-228],[9212,-156],[9396,-248],[9478,-276],[9521,-265]]]},
+{"name":"Caledonia\n(Exclave)","shortname":"Caledonia","color":"#15316a","declutter":3,"x":9450,"z":7850,"id":"56a17dd1-44fb-4277-b43e-15805cf71fa7","polygon":[[[8460,7072],[8820,6888],[9196,6984],[9480,7272],[9752,7464],[9964,7640],[10008,7836],[10332,8044],[10288,8096],[10264,8092],[10240,8156],[10164,8224],[10072,8384],[9852,8636],[9700,8804],[9160,8368],[9016,8252],[8988,8060],[8812,7756],[8692,7492],[8464,7048]]]},
```

Comment:
since it's an exclave: `"shortname":"(Caledonia)"`, to mark that it isn't the "main" caledonia area.

Reply:
Ah makes sense, I'll do that in future.